### ISSUE TITLE
Usage example for SingleOrFallback

### ISSUE THREAD

**Issue by GoogleCodeExporter** [blyzer/morelinq#85]
```
Could you please post a usage example on the documentation of SingleOrFallback 
extension method?

https://code.google.com/p/morelinq/source/browse/MoreLinq/SingleOrFallback.cs

Just tought it would be nice, I sometimes get lost with lambda expressions :S
```

Original issue reported on code.google.com by `ajdvoy...@gmail.com` on 21 Aug 2013 at 5:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 21 Aug 2013 at 6:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 21 Aug 2013 at 6:13
- Added labels: **Component-Docs**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
<example>
<code>
int[] numbers = { 123, 456, 789 };
var result = numbers.Where(x => x == 100).SingleOrFallback(() => {return -1;});
</code>
</example>
```

Original comment by `ajdvoy...@gmail.com` on 7 Oct 2013 at 6:54

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 8 Oct 2013 at 8:59
- Changed state: **Accepted**
- Added labels: **Milestone-Release2.0**

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 8 Oct 2013 at 8:59
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 03f7d8ab05d9.
```

Original comment by `azizatif` on 8 Oct 2013 at 9:01
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 12fcffc7dc51.
```

Original comment by `azizatif` on 12 Aug 2014 at 10:18
